distribution/changelog.txt

```diff
@@ -15,6 +15,7 @@
 - Improved: [#10150] Ride stations are now properly checked if they’re sheltered.
 - Improved: [#10664, #16072] Visibility status can be modified directly in the Tile Inspector's list.
 - Change: [#16077] When importing SV6 files, the RCT1 land types are only added when they were actually used.
+- Fix: [#12774] [Plugin] Plug-ins are now reloaded when New Game option is used in a running game.
```

Comment:
You need to describe the bug, not the fix. This also applies to the commit message.